hydra_redis/querying_mechanism.py

```diff
@@ -526,6 +608,9 @@ def help():
           "show <collection_endpoint> properties")
     print("for classes properties:- show class<class_endpoint> properties")
     print("for compare properties:-show <key> <value> and/or <key1> <value1>")
+    print("for using both opeartions(and,or) you should use brackets like:-",
```

Comment:
Update the commands like this - 
```
Querying format
Get all endpoints :- show endpoints
Get all Class endpoints :- show classEndpoints
Get all Collection endpoints :- show collectionEndpoints
Get all members of a collection endpoint :- show <collection_endpoint> members
Get all properties of an endpoint member :- show object <id_of_member> properties
Get all properties of a class endpoint :- show object <class_endpoint_type> properties
```

Update other commands according to this format as well.

Reply:
okay, I'll update it.